internal/app/repository/repositoryResolver.go

```diff
@@ -0,0 +1,70 @@
+package repository
+
+import (
+	"github.com/jmoiron/sqlx"
+	"go.uber.org/zap"
+	"sanbright/go_shortener/internal/app/entity"
+	"sanbright/go_shortener/internal/config"
+)
+
+var schema = `
+CREATE TABLE IF NOT EXISTS short_link (
+	"uuid" UUID NOT NULL,
+	"short_link" VARCHAR(10) NOT NULL,
+	"url" TEXT NOT NULL,
+	PRIMARY KEY ("uuid")
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS short_link__uniq ON short_link (short_link);
+CREATE UNIQUE INDEX IF NOT EXISTS url__uniq ON short_link (url);
+`
```

Comment:
Можно определить литерах с sql  кодом, как const.
```suggestion
const schema = `
CREATE TABLE IF NOT EXISTS short_link (
	"uuid" UUID NOT NULL,
	"short_link" VARCHAR(10) NOT NULL,
	"url" TEXT NOT NULL,
	PRIMARY KEY ("uuid")
);

CREATE UNIQUE INDEX IF NOT EXISTS short_link__uniq ON short_link (short_link);
CREATE UNIQUE INDEX IF NOT EXISTS url__uniq ON short_link (url);
`
```

 Так же мне кажется правильнее было бы определить схему в  DB Repository, так мы скроем детали реализации и будем соблюдать принцип single responsibility.